PENDING.md

```diff
@@ -38,6 +38,8 @@ IMPROVEMENTS
 
 * Tendermint
 
+ * [\#3699] Upgrade to Tendermint 0.30.1
```

Comment:
This is why I find this process (state machine breaking vs non state machine breaking) to be a bit cumbersome and not seamless -- you can't have the pending update here. You have to wipe the pending log and add a change log entry. Correct me if I'm wrong please (I could very well be).

Reply:
Yes, we need to make the changelog additions there IMO. Thoughts @alessio

Reply:
It's as if we need to maintain two branches per PR...

Reply:
We have to mantain a branch each PR indeed. One goes into develop, the other goes into stable

Reply:
This is because of `PENDING.md` right?

Reply:
@jackzampolin yes. One PR will have pending log entry and the other PR will have a change log entry w/ pending removed.